### PR TITLE
Add grcov test to CI

### LINKED ISSUE #136: Add ccov for this project
Should be similar to what we have here:
https://github.com/mozilla/dump_syms/blob/master/.taskcluster.yml

### PR DESCRIPTION
**Pull request by Luni-4**
This PR fixes #136

### CODE REVIEW

File: .taskcluster.yml [hunk 1]
```diff
@@ -58,6 +58,39 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("grcov_test_task")}
```
Comment (marco-c) [inline .taskcluster.yml:61]: We should integrate code coverage directly in the normal test tasks, this way we don't run the same tests twice.

Comment (Luni-4) [reply .taskcluster.yml:61]: grcov requires the `nightly` version of Rust to be used, while the normal test task could be used to run `rust-code-analyisis` on stable, in this way we test both the versions at the same time

Comment (marco-c) [reply .taskcluster.yml:61]: Sounds good, so let's use stable in the normal test task

File: .taskcluster.yml [hunk 2]
```diff
@@ -58,6 +58,39 @@ tasks:
...
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '1 hour'}
+        provisionerId: proj-relman
+        workerType: ci
+        payload:
+          maxRunTime: 3600
+          image: "rust:buster"
+          env:
+            CODECOV_TOKEN: 1344bbfa-8d56-481e-9cb8-281ba0f9b6af
```
Comment (marco-c) [inline .taskcluster.yml:70]: Let's use the mozilla/rust-code-analysis token: `4df01912-087e-489a-be28-25aa911cb9d2`

File: README.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # rust-code-analysis
 
 [![Task Status](https://community-tc.services.mozilla.com/api/github/v1/repository/mozilla/rust-code-analysis/master/badge.svg)](https://community-tc.services.mozilla.com/api/github/v1/repository/mozilla/rust-code-analysis/master/latest)
+[![codecov](https://codecov.io/gh/calixteman/rust-code-analysis/branch/master/graph/badge.svg)](https://codecov.io/gh/calixteman/rust-code-analysis)
```
Comment (marco-c) [inline README.md:4]:
```suggestion
[![codecov](https://codecov.io/gh/mozilla/rust-code-analysis/branch/master/graph/badge.svg)](https://codecov.io/gh/mozilla/rust-code-analysis)
```

**Pull request merged**